package.json

```diff
@@ -14,7 +14,8 @@
     "postinstall": "prisma generate && husky install",
     "commit": "cz",
     "check-types": "tsc --noEmit --pretty",
-    "cypress:open": "cypress open"
+    "cypress:open": "cypress open",
+    "migrate": "prisma migrate"
```

Comment:
Change the script name and its value to something more semantic. Can be something like: "prisma:migrate": "prisma migrate dev"